app/database/models/connectionsModel.js

```diff
@@ -1,35 +1,16 @@
-/* eslint-disable arrow-parens */
-/* eslint-disable no-console */
 const db = require('../dbConfig.js');
 const Users = require('./usersModel.js');
 
-function getConnections() {
-  return db('connections');
-}
-
 async function getConnectionsByUserId(id) {
-  const conns = await db('connections')
+  const connections = await db('connections')
     .where({ connector_id: id })
     .orWhere({ connected_id: id });
-
-  const ids = [];
-
-  // eslint-disable-next-line no-restricted-syntax
-  for (const conn of conns) {
-    if (!ids.includes(conn.connector_id)) {
-      ids.push(conn.connector_id);
-    }
-    if (!ids.includes(conn.connected_id)) {
-      ids.push(conn.connected_id);
-    }
-  }
-  // TODO possible fix
-  // const ids = Array.from(new Set(conns.flatMap((c) => [c.connector_id, c.connected_id])));
+  const ids = Array.from(new Set(connections.flatMap((c) => [c.connector_id, c.connected_id])));
```

Comment:
can you leave a comment above this explaining that this is to remove duplicates? It would have taken me a minute to parse this if I hadn't seen the deleted code above